admin/load.php

```diff
@@ -634,5 +745,77 @@ function perflab_plugin_admin_notices() {
 		</div>
 		<?php
 	}
+
+	$active_modules_with_plugins = perflab_get_active_modules_with_standalone_plugins();
+
+	if ( ! empty( $active_modules_with_plugins ) ) {
+		$module_data            = perflab_get_modules();
+		$available_module_names = array();
+		foreach ( $active_modules_with_plugins as $module_slug ) {
+			if ( isset( $module_data[ $module_slug ] ) && ! perflab_is_standalone_plugin_loaded( $module_slug ) ) {
+				$available_module_names[] = $module_data[ $module_slug ]['name'];
+			}
+		}
+
+		$modules_count = count( $available_module_names );
+		if ( $modules_count < 1 ) {
+			return;
+		}
+
+		if ( 1 === $modules_count ) {
+			$message  = '<p>';
+			$message .= sprintf(
+				/* translators: Module name */
+				esc_html__( 'Your site is using the "%s" module which will be removed in the future in favor of its equivalent standalone plugin.', 'performance-lab' ),
+				esc_attr( $available_module_names[0] )
+			);
+			$message .= '<br>';
```

Comment:
Is the manual line break needed? On smaller screens, this could result in some unsightly line wrapping. If the issue is that the string is too wide, then a `max-width` on the container would be better.

```suggestion
			$message .= ' ';
```